src/v2/guide/components-slots.md

```diff
@@ -85,145 +163,396 @@ Para estos casos, el elemento `<slot>` tiene un atributo especial, `name`, el cu
 </div>
 ```
 
-Para proporcionar contenido a los slots nombrados, se puede usar el atributo `slot` en un elemento `<template>` en el padre:
+Un `<slot>` sin `nombre` tiene implícitamente el nombre "default".
+
+Para proporcionar contenido a los slots con nombre, podemos usar la directiva `v-slot` en un `<template>`, proporcionando el nombre del slot como argumento de `v-slot`:
 
 ```html
 <base-layout>
-  <template slot="header">
-    <h1>Aquí podría haber un título de página.</h1>
+  <template v-slot:header>
+    <h1>Aquí podría ir un título de página</h1>
   </template>
 
   <p>Un párrafo para el contenido principal.</p>
-  <p>Y otro más.</p>
+  <p>Y otra más. </p>
```

Comment:
I think `otro` was right 🤔 It refers to `párrafo`. What was your thought on chaning it? Maybe I'm missing something.
```suggestion
  <p>Y otro más.</p>
```

Reply:
You are right, no idea why "otra" :)